.github/workflows/metabox.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
           fi
 
   Metabox:
-    if: (github.event.review.state == 'approved' || github.event_name == 'workflow_dispatch') && needs.metabox_run_required.outputs.required_run == 'true'
+    if: (github.event.review.state == 'approved' && needs.metabox_run_required.outputs.required_run == 'true') || github.event_name == 'workflow_dispatch'
```

Comment:
I'm still not sure if this is gonna be enough to trigger this workflow automatically when a PR impacting metabox or checkbox-ng has been approved, but at least now there is an option to trigger it manually.